packages/replay/MIGRATION.md

```diff
@@ -1,3 +1,47 @@
+# End of Replay Beta
+
+Sentry Replay is now out of Beta. This means that the usual stability guarantees apply.
+
+Because of experimentation and rapid iteration, during the Beta period some bugs and problems came up which have since been fixed/improved.
+We **storngly** recommend anyone using Replay in a version before 7.39.0 to update to 7.39.0 or newer, in order to prevent running Replay with known problems that have since been fixed.
```

Comment:
```suggestion
We **strongly** recommend anyone using Replay in a version before 7.39.0 to update to 7.39.0 or newer, in order to prevent running Replay with known problems that have since been fixed.
```